### PR TITLE
Extract SignInForm to new file

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/95?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #83** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/83?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #84** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #87** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/87?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #92** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #93** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/93?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #94** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/94?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #95** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/95?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
                    * **PR #96** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/96?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/95?utm_source=stack-comment).

File: packages/app/features/auth/sign-in-form.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import { ButtonText, H4, Paragraph, SubmitButton, XStack, YStack } from '@my/ui'
+import { useThemeSetting } from '@tamagui/next-theme'
+import { SchemaForm, formFields } from 'app/utils/SchemaForm'
+import { FormProvider, useForm } from 'react-hook-form'
+import { api } from 'app/utils/api'
+import { useRouter } from 'solito/router'
+import { VerifyCode } from './components/VerifyCode'
+import { z } from 'zod'
+
+const SignInSchema = z.object({
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in-form.tsx:10]: would be nice to share this schema with the TRPC route, maybe for another PR.

**Review by 0xBigBoss — APPROVED** [review]
LGTM

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 7, 3:08 AM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/95).

**Pull request merged**